docs/preview/features/endpoint-validation.md

```diff
@@ -97,4 +102,32 @@ The `EventGridAuthorizationAttribute` attribute has some additional consumer-con
 [EventGridAuthorization(..., EmitSecurityEvents = true)]
 ```
 
+## Azure Event Grid subscription validation
+
+This library provides an Azure Event Grid subscription validation. It can receive Azure Event Grid events from an Event subscription and validates the contents.
```

Comment:
We should document if this is supported for Event Grid schema and/or CloudEvents.

Reply:
Well, since we used our `EventParser` and only work with the event payload, it's compatible with both, but this `SubscriptionValidation` event type will probably only be used in Event Grid events, yes?

Reply:
No as the mechanics are different as mentioned below but separate issue it is!

Reply:
https://github.com/arcus-azure/arcus.eventgrid/issues/180